nodejs/src/lib.ts

```diff
@@ -213,13 +213,8 @@ function verify(args: VerifyParameters) {
  * @returns {string} Tl-Signature header kid.
  * @throws {SignatureError} Will throw if signature is invalid.
  */
-function extractKid(tlSignature: string) {
+export function extractKid(tlSignature: string) {
   return parseSignature(tlSignature).headerJson.kid;
 }
 
-export default {
-  sign,
-  verify,
-  extractKid,
-  SignatureError,
-};
+export { SignatureError };
```

Comment:
When I added a plain js usage test I saw that the ts conversion had actually broken import usage from js.

I fixed it so that imports would be unchanged. However, this means in typescript imports will look like
```typescript
import * as tlSigning from 'truelayer-signing';
```

I'm not sure if that's an issue really, but if we want to change it it'll be a breaking change.